### ISSUE TITLE
AI Notebooks - General Availability

### ISSUE THREAD

**Issue by baaastijn** [ovh/public-cloud-roadmap#160]
## User story ##
As a customer, 
I want the ability to launch managed Jupyter or VSCode notebooks with ease and CPU/GPU resources
so that I will focus on my AI exploration jobs and not on the architecture/security layers

## Acceptance criterias ##
- ability to select JupyterLab or VSCode
- ability to select various framworks such as Tensorflow, Pytorch, Huggingface, FastAI, ...
- ability to add labels
- ability to link object storage data
- ability to authorize public access (unsecure access)
- quick link access to JupyterLab or VSCode
- Observability tools (logs and metrics)
- Official support and SLA

**Mockup**
<img width="1043" alt="notebooks" src="https://user-images.githubusercontent.com/16922182/116582049-3d225680-a915-11eb-89b6-bba1a5f88057.png">

## Follow, Vote and give your Feedback ##
You can follow this task with the notification on the right tab.
Ask us anything here in the comments below, and vote with emojis for most requested items !
👍 to vote for this issue

## Discuss on Discord ## 
Feel free to discuss with us on https://discord.com/invite/PwPqWUpN8G

**Comment by baaastijn** [comment]
AI notebooks is now in general availability ! 
Official support and read for production. Prices unchanged. have a good day!
